web-service/README.md

```diff
@@ -1,40 +1,94 @@
+Unlike a UriBeacon, a Physical Web Service is not a core part of the Physical
+Web, is not mandatory, and does not have a specific protocol which must be
+followed (though perhaps an ad-hoc format will arise one day).  It is an
+auxiliary solution to solve a fundamental problem for Physical Web client
+software (see above).
 
-Over time, we've found this to be an elegant way to solve a bunch of other problems for us as well:
+Over time, we've found this to be an elegant way to solve a bunch of other
+problems for us as well:
 * Faster, because we offload the heavy task of content parsing to the server.
 * Cheaper, because we offload the network request costs to the server.
-* Safer, because we can introduce safe-search filtering of inappropriate content.
-* Better, because we can rank results based on various metrics, much as a search engine does.
+* Safer, because we can introduce safe-search filtering of inappropriate
+  content.
+* Better, because we can rank results based on various metrics, much as a
+  search engine does.
 
-## How to run your own Physical Web Service
 
-You can run your own Service by taking a look at the [source code](https://github.com/google/physical-web/tree/master/web-service) and following typical [App Engine deployment documents](https://cloud.google.com/appengine/docs/python/gettingstartedpython27/uploading), or try it locally first by using a [development server](https://cloud.google.com/appengine/docs/python/tools/devserver).  Eventually, you will want to update the `app.yaml` to create a new application ID, and take a look at `config.SAMPLE.json`.
-
-If you are building one of our sample apps and would like to use your own Physical Web Service, you can change the endpoint by modifying:
+## How to run your own Physical Web Service
 
-* Android: Update `METADATA_URL` in [`MetadataResolver.java`](https://github.com/google/physical-web/blob/7c7b5c00f2e6eb08c9730be36a98954334a2e8c6/android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MetadataResolver.java#L48) and `proxy_go_link_base_url` in [`strings.xml`](https://github.com/google/physical-web/blob/7c7b5c00f2e6eb08c9730be36a98954334a2e8c6/android/PhysicalWeb/app/src/main/res/values/strings.xml#L42)
-* iOS: Update `PHYSICALWEB_SERVER_HOSTNAME` in [`PWMetadataRequest.m`](https://github.com/google/physical-web/blob/7c7b5c00f2e6eb08c9730be36a98954334a2e8c6/ios/PhyWeb/Backend/PWMetadataRequest.m#L22)
+You can run your own Service by taking a look at the
+[source code](https://github.com/google/physical-web/tree/master/web-service)
+and following typical
+[App Engine deployment documents](https://cloud.google.com/appengine/docs/python/gettingstartedpython27/uploading),
+or try it locally first by using a
+[development server](https://cloud.google.com/appengine/docs/python/tools/devserver).
+Eventually, you will want to update the `app.yaml` to create a new application
+ID, and take a look at `config.SAMPLE.json`.
```

Comment:
Both files need to be updated manually.  config.SAMPLE.json needs to be updated and renamed to config.SECRET.json.